index.html

```diff
@@ -38,8 +38,16 @@
             </ul>
           
         </nav>
+        
       </header>
-
+      <section class="menuBar">
```

Comment:
- [OPTIONAL ]I think the design and user experience are excellent :clap: but there is one small issue I would like to point out.  I noticed that the visibility of the closing button icon on the mobile version. It was a bit difficult for me to locate the button and close the pop-up window, I believe that if you could make the button more visible in your design, it would greatly improve the user experience for mobile users like me. I can suggest you increase the size, Change the color of the button icon to make it more contrasting against the background, making it easier to locate, and put even some animation or hover effect if possible it will make it more visible and for good UI :wink:  :smile: 

Fix this, please 
![Screenshot from 2023-04-11 10-40-22](https://user-images.githubusercontent.com/88894525/231108208-79593611-08d1-47ed-a737-fdc9778081da.png)